scraper/code_gov/__init__.py

```diff
@@ -156,13 +155,13 @@ def force_attributes(metadata, config):
         logger.debug("Forcing Organization to: %s", organization)
 
     if contact_email:
-        logger.debug("Forcing Contact Email to: %s", contact_email)
+        logger.debug("Forcing Contact Email, as needed, to: %s", contact_email)
 
     for release in metadata["releases"]:
         if organization:
             release["organization"] = organization
 
-        if contact_email:
+        if contact_email and not release["contact"]["email"]:
```

Comment:
@SledKnight - So I know we emailed about this, but after staring at it some more, I'm not sure this IS the right answer... Since we're trying to force the values...

I'm thinking instead there may need to be something more to control if this is done... Soemthing like "set if empty" or such.. I have to parse through the surrounding code a bit though to figure out the best place to do this..

Reply:
Maybe another parameter to control if it is used as an override or default?   "contact_email_default" and "contract_email_override" (instead of the current "contract_email")

Or leave "contract_email" as an override and add "contract_email_default"?  And throw a Warning if one of them is not set?  Have you had any other ideas?  (sorry, I just noticed you had commented on this two weeks ago, as I was checking up on the status of the request today).